drivers/periph_common/gpio.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2019 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_periph_gpio
+ * @{
+ *
+ * @file        gpio.c
+ * @brief       Fallback Implementations for platform specific GPIO routines.
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ * @}
+ */
+
+
+#include "periph/gpio.h"
+
+#ifndef GPIO_HAVE_INIT_LEVEL
```

Comment:
```suggestion
#if !defined(GPIO_HAVE_INIT_LEVEL) || defined(DOXYGEN)
```

Reply:
Does Doxygen care about C files?

Reply:
No. I somehow assumed this to be `static inline` in the header. I was scrolling to fast over the diff :-/